bwc-test/build.gradle

```diff
@@ -47,6 +47,7 @@ buildscript {
         opensearch_version = System.getProperty("opensearch.version", "3.0.0-SNAPSHOT")
         opensearch_group = "org.opensearch"
         common_utils_version = System.getProperty("common_utils.version", '2.9.0.0-SNAPSHOT')
+        jackson_version = System.getProperty("jackson_version", "2.15.2")
```

Comment:
Is it possible to get this referring to core's buildSrc/version.properties?

On the security plugin top-level build.gradlew its done using `${versions.jackson}` like this: https://github.com/opensearch-project/security/blob/main/build.gradle#L425

Reply:
AFAIK these values from from being a plugin, since this is just test code it doesn't have create a plugin where it could be pulled from.